integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -1819,23 +1819,42 @@ fn test_process_block_after_state_sync() {
     genesis.config.epoch_length = epoch_length;
     let mut chain_genesis = ChainGenesis::test();
     chain_genesis.epoch_length = epoch_length;
+
+    let tmp_dir = tempfile::tempdir().unwrap();
```

Comment:
Why we can't reuse `real_epoch_managers` and `nightshade_runtimes` as before?

Reply:
`nightshade_runtimes()` initializes runtime with home_dir `../../../../` where I can't create a new directory. Therefore, I create a runtime manually.
`real_epoch_managers()` by itself is fine, but if I use it, I don't have an epoch manager to pass to the runtime I'm initializing manually.